specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LongTermRetentionPolicies.json

```diff
@@ -243,6 +220,33 @@
           "readOnly": true
         }
       }
+    },
+    "LongTermRetentionPolicyProperties": {
+      "description": "Properties of a long term retention policy",
+      "type": "object",
+      "properties": {
+        "makeBackupsImmutable": {
+          "description": "The setting whether to make LTR backups immutable",
+          "type": "boolean"
+        },
+        "weeklyRetention": {
```

Comment:
>weeklyRetention [](http://example.com/codeflow?start=9&length=15)

consider define "format": "date-time"

Reply:
WeeklyRetention is in the form of PxM where x is the number of weeks.